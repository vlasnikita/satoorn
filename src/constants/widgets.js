import cardIcon from 'Static/widgets/widgets-card.svg'
import lineIcon from 'Static/widgets/widgets-line.svg'

import leftIcon from 'Static/widgets/widgets-invite-left.svg'
import rightIcon from 'Static/widgets/widgets-invite-right.svg'
import topIcon from 'Static/widgets/widgets-invite-top.svg'

import alertIcon from 'Static/widgets/widgets-type-alert.svg'
import inviteIcon from 'Static/widgets/widgets-type-invite.svg'
import challengeIcon from 'Static/widgets/widgets-type-challenge.svg'

import satoornGif from 'Static/widgets/widgets-notification-satoorn.gif'
import giftGif from 'Static/widgets/widgets-notification-gift.gif'
import loveGif from 'Static/widgets/widgets-notification-love.gif'
import popcornGif from 'Static/widgets/widgets-notification-popcorn.gif'

export const CHALLENGE_WIDGET_FORMATS = [
    {
        key: 'card',
        name: 'Карточка',
        icon: cardIcon,
        classnamePostfix: '_card'
    },
    {
        key: 'line',
        name: 'Строка',
        icon: lineIcon,
        classnamePostfix: '_line'

    }
]

export const INVITE_WIDGET_FORMATS = [
    {
        key: 'left',
        icon: leftIcon,
        classnamePostfix: '_left'
    },
    {
        key: 'top',
        icon: topIcon,
        classnamePostfix: '_top'
    },
    {
        key: 'right',
        icon: rightIcon,
        classnamePostfix: '_right'
    }
]

export const NOTIFICATION_WIDGET_ANIMATIONS = [
    {
        key: 'satoorn',
        gif: satoornGif
    },
    {
        key: 'gift',
        gif: giftGif
    },
    {
        key: 'love',
        gif: loveGif
    },
    {
        key: 'popcorn',
        gif: popcornGif
    }
]

export const NOTIFICATION_WIDGET_ALERTS = [
    {
        key: 'newChallenge',
        enabled: true,
        text: 'новый челлендж'
    },
    {
        key: 'newDonation',
        enabled: true,
        text: 'новый донат'
    }
]

export const WIDGET_SETTINGS_DEFAULTS = {
    notifications: {
        animation: NOTIFICATION_WIDGET_ANIMATIONS[0].key,
        sound: 1,
        initialSize:{
            width: 400,
            height: 240
        },
        placeholders: {
            amount: '999',
            username: 'кул_стори_боб',
            title: 'Посмотреть за стрим все тиктоки на свете с закрытыми глазами'
        },
        header: '{amount}₽ - {username}!',
        body: '{title} - {link}'
    },
    invite: {
        format: INVITE_WIDGET_FORMATS[0].key,
        text: 'создай мне челлендж',
        isTransparent: true,
        initialSize:{
            width: 360,
            height: 180
        }
    },
    challenge: {
        format: CHALLENGE_WIDGET_FORMATS[0].key,
        initialSize:{
            width: 240,
            height: 190
        }
    }
}

export const WIDGET_TYPES = [
    {
        key: 'notifications',
        name: 'Алерты и оповещения',
        icon: alertIcon,
        height: 22
    },
    {
        key: 'invite',
        name: 'Инвайт-виджет',
        icon: inviteIcon,
        height: 18
    },
    {
        key: 'challenge',
        name: 'Челленджи',
        icon: challengeIcon,
        height: 22
    }
]