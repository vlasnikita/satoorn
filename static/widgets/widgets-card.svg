<svg width="62" height="49" viewBox="0 0 62 49" fill="none" xmlns="http://www.w3.org/2000/svg">
<rect width="62" height="49" fill="white"/>
<rect width="61.9835" height="6.32191" fill="white"/>
<rect x="2.47656" y="31.7476" width="52.0661" height="3.79315" rx="1.89657" fill="#EEEEEE" fill-opacity="0.35"/>
<rect x="2.47656" y="36.8052" width="47.1074" height="3.79315" rx="1.89657" fill="#EEEEEE" fill-opacity="0.35"/>
<rect x="2.47656" y="41.8633" width="49.5868" height="3.79315" rx="1.89657" fill="#EEEEEE" fill-opacity="0.35"/>
<ellipse cx="6.81931" cy="4.42534" rx="4.33884" ry="4.42534" fill="white"/>
<rect y="6" width="62" height="26" fill="white"/>
<rect x="3" y="9" width="56" height="20" rx="1" fill="#F2D20B"/>
<rect x="10" y="4" width="32" height="4" rx="2" fill="#EEEEEE" fill-opacity="0.35"/>
<circle cx="8" cy="6" r="4.75" fill="#EEEEEE" stroke="white" stroke-width="0.5"/>
<path d="M24.6816 22.0527H23.1289V23H21.6641V22.0527H20.6729V20.8662H21.6641V20.4951H20.6729V19.3086H21.6641V15.8906H24.4277C24.9616 15.8906 25.4303 15.9867 25.834 16.1787C26.2376 16.3708 26.5501 16.6458 26.7715 17.0039C26.9961 17.3587 27.1084 17.764 27.1084 18.2197C27.1084 18.9229 26.8691 19.4779 26.3906 19.8848C25.9154 20.2917 25.2529 20.4951 24.4033 20.4951H23.1289V20.8662H24.6816V22.0527ZM23.1289 19.3086H24.3936C25.2139 19.3086 25.624 18.9489 25.624 18.2295C25.624 17.8877 25.5199 17.6126 25.3115 17.4043C25.1032 17.1927 24.8151 17.0853 24.4473 17.082H23.1289V19.3086ZM31.7324 22.0527H30.1797V23H28.7148V22.0527H27.7236V20.8662H28.7148V20.4951H27.7236V19.3086H28.7148V15.8906H31.4785C32.0124 15.8906 32.4811 15.9867 32.8848 16.1787C33.2884 16.3708 33.6009 16.6458 33.8223 17.0039C34.0469 17.3587 34.1592 17.764 34.1592 18.2197C34.1592 18.9229 33.9199 19.4779 33.4414 19.8848C32.9661 20.2917 32.3037 20.4951 31.4541 20.4951H30.1797V20.8662H31.7324V22.0527ZM30.1797 19.3086H31.4443C32.2646 19.3086 32.6748 18.9489 32.6748 18.2295C32.6748 17.8877 32.5706 17.6126 32.3623 17.4043C32.154 17.1927 31.8659 17.0853 31.498 17.082H30.1797V19.3086ZM38.7832 22.0527H37.2305V23H35.7656V22.0527H34.7744V20.8662H35.7656V20.4951H34.7744V19.3086H35.7656V15.8906H38.5293C39.0632 15.8906 39.5319 15.9867 39.9355 16.1787C40.3392 16.3708 40.6517 16.6458 40.873 17.0039C41.0977 17.3587 41.21 17.764 41.21 18.2197C41.21 18.9229 40.9707 19.4779 40.4922 19.8848C40.0169 20.2917 39.3545 20.4951 38.5049 20.4951H37.2305V20.8662H38.7832V22.0527ZM37.2305 19.3086H38.4951C39.3154 19.3086 39.7256 18.9489 39.7256 18.2295C39.7256 17.8877 39.6214 17.6126 39.4131 17.4043C39.2048 17.1927 38.9167 17.0853 38.5488 17.082H37.2305V19.3086Z" fill="white"/>
</svg>
