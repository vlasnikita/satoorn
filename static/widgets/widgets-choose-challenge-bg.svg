<svg width="128" height="127" viewBox="0 0 128 127" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M100.572 55.168L113.605 45.699H97.4955C97.0195 45.699 96.5975 45.3925 96.4505 44.9397L91.4725 29.6191L86.4945 44.9397C86.3474 45.3927 85.9255 45.699 85.4494 45.699H69.3398L82.3727 55.168C82.7578 55.4478 82.9191 55.9438 82.7719 56.3966L77.7939 71.7176L90.8263 62.2486C91.2116 61.9688 91.7329 61.9688 92.1182 62.2486L105.151 71.7176L100.173 56.3966C100.026 55.9438 100.187 55.448 100.572 55.168Z" fill="#E6E6F0"/>
<path d="M93.165 103.884C91.4568 103.262 89.5613 104.146 88.9396 105.854C88.3179 107.562 89.2018 109.458 90.9099 110.079C92.6181 110.701 94.5136 109.817 95.1353 108.109C95.7571 106.401 94.8732 104.505 93.165 103.884Z" fill="#E6E6F0"/>
<path d="M43.3344 31.9537C45.6117 32.7825 48.1393 31.6041 48.9682 29.3265C49.7972 27.049 48.6185 24.5216 46.3411 23.6927C44.0638 22.8638 41.5362 24.0422 40.7072 26.3198C39.8783 28.5973 41.057 31.1248 43.3344 31.9537Z" fill="#E6E6F0"/>
<path d="M68.0671 78.719C67.9389 78.444 67.7112 78.247 67.4468 78.1508C67.1827 78.0547 66.8817 78.0592 66.6067 78.1874L38.4374 91.323C37.8872 91.5796 37.6492 92.2334 37.9059 92.7833C38.1625 93.3333 38.8159 93.5711 39.366 93.3148L67.5353 80.1792C68.0854 79.9229 68.3234 79.269 68.0671 78.719Z" fill="#E6E6F0"/>
<path d="M68.4446 64.8247C68.3165 64.5496 68.0888 64.3526 67.8245 64.2564C67.5603 64.1602 67.2593 64.1648 66.9843 64.2932L14.8711 88.5938C14.3209 88.8504 14.0829 89.5042 14.3395 90.0542C14.5962 90.6041 15.2496 90.8419 15.7997 90.5856L67.913 66.2848C68.463 66.0284 68.701 65.3746 68.4446 64.8247Z" fill="#E6E6F0"/>
<path d="M32.2086 68.0072C32.0802 67.7322 31.8527 67.5353 31.5885 67.4391C31.3244 67.343 31.0234 67.3476 30.7484 67.4758L13.8468 75.3571C13.2967 75.6137 13.0587 76.2675 13.3153 76.8175C13.5719 77.3674 14.2254 77.6052 14.7754 77.3489L31.677 69.4675C32.2271 69.2112 32.4651 68.5574 32.2086 68.0072Z" fill="#E6E6F0"/>
<path d="M10.88 64.2357C11.1366 64.7856 11.7901 65.0234 12.3401 64.7671L63.045 41.1231C63.595 40.8667 63.833 40.2129 63.5764 39.6629C63.3199 39.1128 62.6664 38.875 62.1163 39.1315L11.4115 62.7753C10.8614 63.0319 10.6234 63.6857 10.88 64.2357Z" fill="#E6E6F0"/>
<path d="M87.4503 78.7582L59.2809 91.8937C58.7308 92.1503 58.4928 92.8041 58.7494 93.3541C59.006 93.904 59.6597 94.1419 60.2098 93.8856L88.3791 80.75C88.9292 80.4937 89.1672 79.8398 88.9106 79.2897C88.654 78.7397 88.0003 78.502 87.4503 78.7582Z" fill="#E6E6F0"/>
<path d="M34.0202 92.8815C32.8796 92.4663 31.6184 93.0544 31.2032 94.195C30.7881 95.3357 31.3762 96.5968 32.5168 97.012C33.6574 97.4271 34.9186 96.839 35.3337 95.6984C35.7489 94.5578 35.1608 93.2966 34.0202 92.8815Z" fill="#E6E6F0"/>
<path d="M40.4147 62.465C39.2741 62.0498 38.0129 62.6379 37.5978 63.7785C37.1826 64.9191 37.7707 66.1803 38.9113 66.5955C40.0519 67.0106 41.3131 66.4225 41.7283 65.2819C42.1434 64.1413 41.5553 62.8801 40.4147 62.465Z" fill="#E6E6F0"/>
<path d="M68.942 112.609C67.8014 112.193 66.5402 112.781 66.1251 113.922C65.71 115.063 66.2981 116.324 67.4387 116.739C68.5793 117.154 69.8404 116.566 70.2556 115.425C70.6707 114.285 70.0826 113.024 68.942 112.609Z" fill="#E6E6F0"/>
<path d="M98.2428 83.5128C97.1022 83.0977 95.841 83.6858 95.4259 84.8264C95.0107 85.967 95.5988 87.2282 96.7394 87.6433C97.88 88.0585 99.1412 87.4704 99.5564 86.3298C99.9715 85.1892 99.3834 83.928 98.2428 83.5128Z" fill="#E6E6F0"/>
<path d="M28.5944 37.1124C27.4538 36.6973 26.1926 37.2854 25.7774 38.426C25.3623 39.5666 25.9504 40.8278 27.091 41.2429C28.2316 41.6581 29.4928 41.07 29.9079 39.9294C30.3231 38.7888 29.735 37.5276 28.5944 37.1124Z" fill="#E6E6F0"/>
<path d="M68.5905 23.6051C67.4499 23.19 66.1887 23.7781 65.7735 24.9187C65.3584 26.0593 65.9465 27.3205 67.0871 27.7356C68.2277 28.1508 69.4889 27.5627 69.904 26.4221C70.3192 25.2815 69.7311 24.0203 68.5905 23.6051Z" fill="#E6E6F0"/>
<path d="M120.786 47.279C119.645 46.8638 118.384 47.4519 117.969 48.5925C117.554 49.7331 118.142 50.9943 119.282 51.4094C120.423 51.8246 121.684 51.2365 122.099 50.0959C122.514 48.9553 121.926 47.6941 120.786 47.279Z" fill="#E6E6F0"/>
</svg>
