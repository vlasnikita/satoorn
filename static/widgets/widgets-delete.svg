<svg xmlns:x="http://ns.adobe.com/Extensibility/1.0/" xmlns:i="http://ns.adobe.com/AdobeIllustrator/10.0/"
     xmlns:graph="http://ns.adobe.com/Graphs/1.0/" xmlns="http://www.w3.org/2000/svg"
     xmlns:xlink="http://www.w3.org/1999/xlink" height="22px" width="22px" fill="#888" version="1.1" x="0px"
     y="0px" viewBox="0 0 100 100" style="enable-background:new 0 0 100 100;" xml:space="preserve"><switch><foreignObject requiredExtensions="http://ns.adobe.com/AdobeIllustrator/10.0/" x="0" y="0" width="1" height="1"/><g
        i:extraneous="self"><g><path d="M33.2,80.7c1.5,0,2.8-1.3,2.8-2.8V47.2c0-1.5-1.3-2.8-2.8-2.8c-1.5,0-2.8,1.3-2.8,2.8v30.7     C30.4,79.5,31.7,80.7,33.2,80.7z"/><path
        d="M50,80.7c1.5,0,2.8-1.3,2.8-2.8V47.2c0-1.5-1.3-2.8-2.8-2.8c-1.5,0-2.8,1.3-2.8,2.8v30.7C47.2,79.5,48.5,80.7,50,80.7z"/><path
        d="M66.8,80.7c1.5,0,2.8-1.3,2.8-2.8V47.2c0-1.5-1.3-2.8-2.8-2.8c-1.5,0-2.8,1.3-2.8,2.8v30.7C64,79.5,65.2,80.7,66.8,80.7z     "/><path
        d="M86.3,13.7H64v-2.8c0-4.6-3.8-8.4-8.4-8.4H44.4c-4.6,0-8.4,3.8-8.4,8.4v2.8H13.7c-4.6,0-8.4,3.8-8.4,8.4v2.8     c0,4.6,3.8,8.4,8.4,8.4v55.9c0,4.6,3.8,8.4,8.4,8.4h55.9c4.6,0,8.4-3.8,8.4-8.4V33.2c4.6,0,8.4-3.8,8.4-8.4v-2.8     C94.7,17.4,90.9,13.7,86.3,13.7z M41.6,10.9c0-1.5,1.3-2.8,2.8-2.8h11.2c1.5,0,2.8,1.3,2.8,2.8v2.8H41.6V10.9z M80.7,89.1     c0,1.5-1.3,2.8-2.8,2.8H22.1c-1.5,0-2.8-1.3-2.8-2.8V33.2h61.5V89.1z M89.1,24.9c0,1.5-1.3,2.8-2.8,2.8H13.7     c-1.5,0-2.8-1.3-2.8-2.8v-2.8c0-1.5,1.3-2.8,2.8-2.8h72.6c1.5,0,2.8,1.3,2.8,2.8V24.9z"/></g></g></switch></svg>