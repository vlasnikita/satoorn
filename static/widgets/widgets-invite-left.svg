<svg width="24" height="7" viewBox="0 0 24 7" fill="none" xmlns="http://www.w3.org/2000/svg">
<circle cx="3.5" cy="3.5" r="3.5" fill="#FEE233"/>
<line x1="9" y1="2" x2="20" y2="2" stroke="#DADADA" stroke-width="2" stroke-linecap="round"/>
<line x1="9" y1="5" x2="23" y2="5" stroke="#DADADA" stroke-width="2" stroke-linecap="round"/>
</svg>
