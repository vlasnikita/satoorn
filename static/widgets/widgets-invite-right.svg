<svg width="24" height="7" viewBox="0 0 24 7" fill="none" xmlns="http://www.w3.org/2000/svg">
<circle cx="20.5" cy="3.5" r="3.5" transform="rotate(-180 20.5 3.5)" fill="#FEE233"/>
<line x1="15" y1="5" x2="4" y2="5" stroke="#DADADA" stroke-width="2" stroke-linecap="round"/>
<line x1="15" y1="2" x2="1" y2="2" stroke="#DADADA" stroke-width="2" stroke-linecap="round"/>
</svg>
