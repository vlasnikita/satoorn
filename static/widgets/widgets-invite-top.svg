<svg width="16" height="13" viewBox="0 0 16 13" fill="none" xmlns="http://www.w3.org/2000/svg">
<circle cx="8" cy="3.5" r="3.5" fill="#FEE233"/>
<line x1="2.5" y1="9" x2="13.5" y2="9" stroke="#DADADA" stroke-width="2" stroke-linecap="round"/>
<line x1="1" y1="12" x2="15" y2="12" stroke="#DADADA" stroke-width="2" stroke-linecap="round"/>
</svg>
