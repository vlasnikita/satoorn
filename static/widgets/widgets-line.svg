<svg width="117" height="31" viewBox="0 0 117 31" fill="none" xmlns="http://www.w3.org/2000/svg">
<rect width="117" height="31" fill="white"/>
<rect width="117" height="31" rx="1" fill="white"/>
<rect x="18" y="5" width="53" height="8" rx="3" fill="#EEEEEE" fill-opacity="0.35"/>
<rect x="8" y="19" width="94" height="4" rx="2" fill="#EEEEEE" fill-opacity="0.35"/>
<rect x="8" y="24" width="81" height="4" rx="2" fill="#EEEEEE" fill-opacity="0.35"/>
<circle cx="13" cy="9" r="7.5" fill="#EEEEEE" stroke="white"/>
<rect x="76.8359" y="2.16699" width="37.2452" height="13.833" rx="2.5" fill="#EEEEEE"/>
<rect x="78.082" y="3.03076" width="34.7622" height="12.1039" rx="2" fill="#F2D20B"/>
<path d="M90.4453 11.2422H89.2031V12H88.0312V11.2422H87.2383V10.293H88.0312V9.99609H87.2383V9.04688H88.0312V6.3125H90.2422C90.6693 6.3125 91.0443 6.38932 91.3672 6.54297C91.6901 6.69661 91.9401 6.91667 92.1172 7.20312C92.2969 7.48698 92.3867 7.8112 92.3867 8.17578C92.3867 8.73828 92.1953 9.18229 91.8125 9.50781C91.4323 9.83333 90.9023 9.99609 90.2227 9.99609H89.2031V10.293H90.4453V11.2422ZM89.2031 9.04688H90.2148C90.8711 9.04688 91.1992 8.75911 91.1992 8.18359C91.1992 7.91016 91.1159 7.6901 90.9492 7.52344C90.7826 7.35417 90.5521 7.26823 90.2578 7.26562H89.2031V9.04688ZM96.0859 11.2422H94.8438V12H93.6719V11.2422H92.8789V10.293H93.6719V9.99609H92.8789V9.04688H93.6719V6.3125H95.8828C96.3099 6.3125 96.6849 6.38932 97.0078 6.54297C97.3307 6.69661 97.5807 6.91667 97.7578 7.20312C97.9375 7.48698 98.0273 7.8112 98.0273 8.17578C98.0273 8.73828 97.8359 9.18229 97.4531 9.50781C97.0729 9.83333 96.543 9.99609 95.8633 9.99609H94.8438V10.293H96.0859V11.2422ZM94.8438 9.04688H95.8555C96.5117 9.04688 96.8398 8.75911 96.8398 8.18359C96.8398 7.91016 96.7565 7.6901 96.5898 7.52344C96.4232 7.35417 96.1927 7.26823 95.8984 7.26562H94.8438V9.04688ZM101.727 11.2422H100.484V12H99.3125V11.2422H98.5195V10.293H99.3125V9.99609H98.5195V9.04688H99.3125V6.3125H101.523C101.951 6.3125 102.326 6.38932 102.648 6.54297C102.971 6.69661 103.221 6.91667 103.398 7.20312C103.578 7.48698 103.668 7.8112 103.668 8.17578C103.668 8.73828 103.477 9.18229 103.094 9.50781C102.714 9.83333 102.184 9.99609 101.504 9.99609H100.484V10.293H101.727V11.2422ZM100.484 9.04688H101.496C102.152 9.04688 102.48 8.75911 102.48 8.18359C102.48 7.91016 102.397 7.6901 102.23 7.52344C102.064 7.35417 101.833 7.26823 101.539 7.26562H100.484V9.04688Z" fill="white"/>
</svg>
