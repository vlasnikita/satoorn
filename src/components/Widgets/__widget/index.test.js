import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import { WIDGET_SETTINGS_DEFAULTS, WIDGET_TYPES } from 'Constants/widgets'
import Widget from "./index";

it('should render Widget correctly', () => {
    const props = {
        type: WIDGET_TYPES[0].key,
        settings: WIDGET_SETTINGS_DEFAULTS[WIDGET_TYPES[0].key],
        position: {x: 0, y: 0},
        size: WIDGET_SETTINGS_DEFAULTS[WIDGET_TYPES[0].key].initialSize,
        activeStreamer: {
            login: 'some Login'
        }
    }
    const renderer = new ShallowRenderer();
    const wrapper = renderer.render(<Widget {...props} />)
    expect(wrapper).toMatchSnapshot()
});