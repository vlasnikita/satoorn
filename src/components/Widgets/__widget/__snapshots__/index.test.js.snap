// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Widget correctly 1`] = `
<Rnd
  bounds=".Widgets__canvas"
  className="Widgets__widgetContainer "
  lockAspectRatio={true}
  maxHeight={9007199254740991}
  maxWidth={9007199254740991}
  onDrag={[Function]}
  onDragStart={[Function]}
  onDragStop={[Function]}
  onResize={[Function]}
  onResizeStart={[Function]}
  onResizeStop={[Function]}
  position={
    Object {
      "x": 0,
      "y": 0,
    }
  }
  resizeHandleClasses={
    Object {
      "bottom": "Widgets__widgetResize",
      "bottomLeft": "Widgets__widgetResize",
      "bottomRight": "Widgets__widgetResize",
      "left": "Widgets__widgetResize",
      "right": "Widgets__widgetResize",
      "top": "Widgets__widgetResize",
      "topLeft": "Widgets__widgetResize",
      "topRight": "Widgets__widgetResize",
    }
  }
  scale={1}
  size={
    Object {
      "height": 240,
      "width": 400,
    }
  }
>
  <div
    className="
                    Widgets__widget 
                    Widgets__widget_notifications"
    style={
      Object {
        "height": 240,
      }
    }
  >
    <img
      className="Widgets__widgetAnimation"
      src="widgets-notification-satoorn.gif"
      style={
        Object {
          "height": 160,
        }
      }
    />
    <p
      className="Widgets__widgetMessage Widgets__widgetMessage_header"
      dangerouslySetInnerHTML={
        Object {
          "__html": "999₽ - кул_стори_боб!",
        }
      }
      style={
        Object {
          "fontSize": 22.8,
          "marginBottom": 4.8,
        }
      }
    />
    <p
      className="Widgets__widgetMessage"
      dangerouslySetInnerHTML={
        Object {
          "__html": "Посмотреть за стрим все тиктоки на свете с закрытыми глазами - <span class=\\"Widgets__widgetMessage_link\\">satoorn.ru/some Login</span>",
        }
      }
      style={
        Object {
          "fontSize": 12.4,
        }
      }
    />
  </div>
  <button
    className="Widgets__widgetDelete"
    onClick={[Function]}
  >
    <img
      alt=""
      src="widgets-delete.svg"
    />
  </button>
</Rnd>
`;
